### DISCUSSION TITLE
Turn Off Screen At Night ?

### DISCUSSION THREAD

**Discussion by Cjdavidson** [HASwitchPlate/HASPone#4] [Q&A] [answered]
I have a HASPone in my bedroom and at night its is so bright at 2 or 1 brightness.
Is it possible to have the screen backlight go off at night at a set time and come on in the morning or a set time. And have it so if i get up in the night i touch the screen at it comes on at brightness 20 for about 10 secs then go off again until the morning.

Thanks for all your great work
Cameron

**Comment by aderusha** [comment, marked as answer]
A nice feature of HASPone having better Home Assistant integration is that blueprints made by others to control things like lights will work with your HASPone!

For example - I suspect this guy will accomplish most of what you're after: https://community.home-assistant.io/t/dim-lights-based-on-the-suns-elevation/269701

However, I don't think it will handle "turn the screen back full brightness on interaction", so maybe I need to take a look at some additional options there. 

Thanks for the suggestion!

**Reply by Cjdavidson** [reply]
Thanks I would appreciate it and my wife.
